### PR TITLE
tests added

### PR DESCRIPTION
**Pull request by mjsheikh**

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -24,7 +24,7 @@ if GROUP == "All" || GROUP == "Interface"
     @time @safetestset "2nd order check" begin include("2nd_order_check.jl") end
     @time @safetestset "Non-linear Diffusion" begin include("Fast_Diffusion.jl") end
     #@time @safetestset "KdV" begin include("KdV.jl") end # KdV times out and all fails
```
Comment (ChrisRackauckas) [inline test/runtests.jl:26]: is there a reason why KdV wasn't added back?

Comment (mjsheikh) [reply test/runtests.jl:26]: Because tests would be broken for the 2-Soliton case

Comment (ChrisRackauckas) [reply test/runtests.jl:26]: just mark that section?

Comment (mjsheikh) [reply test/runtests.jl:26]: If you say so :D

**Pull request merged**